packages/modules/auth/src/services/auth-provider.ts

```diff
@@ -26,10 +25,11 @@ export default class AuthProviderService {
     try {
       return this.dependencies[`${AuthProviderRegistrationPrefix}${providerId}`]
     } catch (err) {
-      throw new MedusaError(
-        MedusaError.Types.NOT_FOUND,
-        `Could not find a auth provider with id: ${providerId}`
-      )
+      const errMessage = `
+      Unable to retreieve the auth provider with id: ${providerId}
+      Please make sure that the provider is registered in the container and it is configured correctly in your project configuration file.
+      `
+      throw new Error(errMessage)
```

Comment:
question: Just wanna make sure the change from `MedusaError` to `Error` intentional?

Reply:
Nvm, I see its added throughout the PR, so I guess it is 😄

Reply:
yes it is :), I dont think it has to be a medusa error here but if you think otherwise I would be happy to change it

Reply:
Fine to keep it like this!